ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -2,7 +2,9 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { Tabs, Tab } from '../../../ui/tabs';
+///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
```

Comment:
Those type of fencing will be a problem if someone wants to add a new build type and be confused why it's not building normally.

It's not a stopper and fine to merge as is, but if you have the time, you could consider if a better solution exists.

* One would be to have `ONLY_EXCLUDE_IN` directive in [remove-fenced-code.js](https://github.com/MetaMask/metamask-extension/blob/37b481f3056344de6d9c763ebeeaccc18146642e/development/build/transforms/remove-fenced-code.js#L124)
* Another one would be have an empty feature such as `group-not-mmi`
* Or just include Button in MMI

Reply:
@brad-decker @zone-live wdyt?